scratch/PCA/extract_data.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
Once a file has been added to a repo, git will track it regardless of what is in the `.gitignore`; i.e. the `.gitignore` will only have an effect on new files that are not yet under version control.  To remove the file now, use `git rm`